woocommerce/woocommerce
# CYS - Make the patterns content translatable

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/49224



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `wp-admin/options-general.php` and change the language to something different than `English`.
2. Open the [PatternsDictionary.php](https://github.com/woocommerce/woocommerce/blob/bb2d0bb402f5b6a63341648d187d8222e0d47976/plugins/woocommerce/src/Blocks/AIContent/PatternsDictionary.php) file and replace some string for something that has translations. For example, change the `Home decor` in the `Featured Category Triple` to `Shop` or `Shop now`.
3. Go to `wp-admin/admin.php?page=wc-admin&path=%2Fcustomize-store` and click `Start designing` or `Customize your store` to go to the assembler.
4. Click on `Design your homepage`.
5. Click the category corresponding to the pattern you modified (in this example, click `Featured selling`).
6. Insert the pattern (in this example, `Featured Category Triple`).
7. Check the string you modified is translated.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
CYS - Make the patterns content translatable for patterns in the dictionary.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
